connectors/uplynk/src/main/java/com/theoplayer/android/connector/uplynk/UplynkConnector.kt

```diff
@@ -23,7 +26,52 @@ class UplynkConnector(
     private val theoplayerView: THEOplayerView,
 ) {
     private lateinit var integration: UplynkAdIntegration
-    private val eventDispatcher = UplynkEventDispatcher()
+
+    /**
+     * Scheduled ad breaks
+     */
+    var adBreaks: List<UplynkAdBreak>? = null
+        private set
+
+    /**
+     * The ad break that is currently played or `null` otherwise
+     */
+    var currentAdBreak: UplynkAdBreak? = null
+        private set
+
+    /**
+     * The ad that is currently played or `null` otherwise
+     */
+    var currentAd: UplynkAd? = null
+        private set
+
+    private val eventDispatcher = UplynkEventDispatcher().also {
+        it.addListener(object : UplynkListener {
+            override fun onAdBreaksUpdated(ads: UplynkAds) {
+                this@UplynkConnector.adBreaks = ads.breaks
```

Comment:
Nit-pick: is the `this@UplynkConnector` qualifier always necessary? I would expect you can just write:
```suggestion
                adBreaks = ads.breaks
```

Reply:
No, it's not necessary. It is a leftover from some previous version of my code. Thanks!

Reply:
Done